cmd/cmd.go

```diff
@@ -63,13 +68,18 @@ func AddFlags(cmd *cobra.Command) {
 		"Set the log file path. If not set, logs will output to temp file")
 	cmd.PersistentFlags().String(FlagLogFormat, "text",
 		"Set the log format")
+	cmd.PersistentFlags().Bool(FlagRedactLog, false,
+		"Set whether to redact sensitive info in log, already deprecated by --redact-info-log")
```

Comment:
did someone said --redact-log was included in 4.0.10???

Reply:
Let's keep it (for the compatibility with 5.0.0-rc). 😥